Privacy.md

```diff
@@ -0,0 +1,88 @@
+
+Forecastie needs this permission below;
+
+- Location
+
+With the location permission enabled, the app can;
+
+- access approximate location (network-based).
+- access precise location (GPS and network-based).
+
+Forecastie has some other capabilities, which are functions the software can remotely effect on its own. They include;
+
+- **have full network access:** The app is capable of creating network sockets and can use customized network protocols. This capability does not require the software to send data to the internet.
+- **view network connections:** This is the capability of the app to view information about network connections such as which networks exist and are connected.
+- **run at startup:** Allows the app to have itself started as soon as your device has finished booting.
+
+## Types of Data Collected
+
+### Personal Data
+The developer doesn't collect personally identifiable data.
+
+### Usage Data
+The developer doesn't collect usage data.
+
+### Installation Data
+The developer distributes this software via channels like F-Droid and GitHub where download counts are not maintained. So the developer has no knowledge or way to acquire knowledge that the user has installed or is using this software. The developer also distributes this software via google play store, but the developer is not privy to this information either.
+
+
+## Crash reports sending
+
+Forecastie is stable and works fine, but you may still experience a few errors and crashes. The information about these crashes is important to the developer so the issues can be fixed immediately and the software is running efficiently again. The developer isn't able to get reports when the app crashes. If you encounter a bug during the use of this software, please open an [issue](https://github.com/martykan/forecastie/issues/new) on GitHub and it will be fixed in due time. When submitting a crash report, please provide a logcat to help the developer pinpoint the cause of the crash quickly and fix it accordingly.
+
+
+## Cookies
+
+Cookies are files with a small amount of data that are commonly used as anonymous unique identifiers. These are sent to your browser from the websites that you visit and are stored on your device's internal memory.
+
+This Service does not use these “cookies” explicitly. However, the app may use third party code and libraries that use “cookies” to collect information and improve their services. You have the option to either accept or refuse these cookies and know when a cookie is being sent to your device. If you choose to refuse our cookies, you may not be able to use some portions of this Service.
```

Comment:
The word "Our" towards the end is misleading. Are they our cookies, or those provided by third parties?